website/content/docs/templates/hcl_templates/contextual-variables.mdx

```diff
@@ -145,6 +147,22 @@ source "amazon-ebs" "cannonical-ubuntu-server" {
 }
 ```
 
+# HCP Packer Version Fingerprint
+
+If your build is pushing metadata to the HCP Packer registry, this variable is
+set to the value of the Version Fingerprint associated with this run.
+
+```hcl
+source "amazon-ebs" "cannonical-ubuntu-server" {
+  ami_name         = "packer-example"
+  // ...
+  run_volume_tags = {
+    hcp_version_fingerprint = packer.versionFingerprint
+  }
+}
+```
+
+
 ```shell-session
 ==> vanilla.amazon-ebs.cannonical-ubuntu-server: Adding tags to source instance
     vanilla.amazon-ebs.cannonical-ubuntu-server: Adding tag: "Name": "Packer Builder"
```

Comment:
We should update the logs here so they don't show the `hcp_iteration_id` in there right?